Sources/Commands/Utilities/APIDigester.swift

```diff
@@ -145,31 +145,34 @@ struct APIDigesterBaselineDumper {
             toolsBuildParameters: toolsBuildParameters,
             packageGraphLoader: { graph }
         )
-        try buildSystem.build()
+        try await buildSystem.build()
 
         // Dump the SDK JSON.
         try swiftCommandState.fileSystem.createDirectory(baselineDir, recursive: true)
-        let group = DispatchGroup()
-        let semaphore = DispatchSemaphore(value: Int(productsBuildParameters.workers))
-        let errors = ThreadSafeArrayStore<Swift.Error>()
-        for module in modulesToDiff {
-            semaphore.wait()
-            DispatchQueue.sharedConcurrent.async(group: group) {
-                do {
-                    try apiDigesterTool.emitAPIBaseline(
-                        to: baselinePath(module),
-                        for: module,
-                        buildPlan: buildSystem.buildPlan
-                    )
-                } catch {
-                    errors.append(error)
+
+
+        // TODO: Do we need to support worker paramater?
```

Comment:
Formatting and typo nit. Also, can you elaborate on the `TODO`, what's the parameter you're referring to here?
```suggestion

        // TODO: Do we need to support worker parameter?
```